Non-charring/PMMA/UMD/UMD_CAPA_60kW_Mass.csv

```diff
@@ -1,74 +1,75 @@
-Mean Time [s],Mean MLR [kg/s/m^2],MLR_error [kg/s/m^2]
-2.49315,-0.000260366,0.001030198
-7.50806,-0.0002508,0.000454634
-12.5234475,0.001199156,0.000289296
-17.5398075,0.000973333,0.000337859
-22.55627,0.004774699,0.000927216
-27.57311,0.01073036,0.001089449
-32.591445,0.015633281,0.00069556
-37.6117575,0.017238126,0.00058606
-42.6298425,0.01814478,0.000399137
-47.64548,0.018913583,0.000304719
-52.6607175,0.020564919,0.000364624
-57.6753825,0.0209623,0.000343216
-62.56814474,0.022633181,0.000691289
-67.45539,0.02250719,0.000585457
-72.4715,0.022839544,0.00071374
-77.487735,0.023046238,0.000412467
-82.5044,0.022196645,0.000592869
-87.518615,0.023085009,0.000342244
-92.53275,0.023889451,0.000370589
-97.54921,0.024100855,0.000748653
-102.5644725,0.023410945,0.000630322
-107.4496,0.023942394,0.0007359
-112.342405,0.024455283,0.000342419
-117.3600675,0.024295505,0.000397993
-122.3773325,0.024083619,0.000545008
-127.3943675,0.02500547,0.000452827
-132.4092275,0.024398757,0.000494299
-137.42334,0.025505756,0.000412429
-142.439375,0.025378656,0.000678547
-147.455615,0.025327273,0.000354926
-152.47018,0.024877874,0.00043408
-157.48514,0.026503795,0.000568927
-162.5021,0.026260242,0.00040276
-167.518485,0.026796176,0.000324452
-172.5329975,0.02711086,0.000432136
-177.5484125,0.027360529,0.000420058
-182.5651,0.02743618,0.000432462
-187.581285,0.028326356,0.000528299
-192.5992225,0.028500382,0.000493924
-197.6157325,0.028593994,0.000610944
-202.6320175,0.030100971,0.000446942
-207.6474275,0.029752516,0.000407067
-212.6632425,0.030082479,0.000738712
-217.67848,0.030638814,0.000618145
-222.6913425,0.031090109,0.000573707
-227.5850289,0.031735782,0.000806875
-232.4709025,0.031816067,0.000683193
-237.4857175,0.033612668,0.000683338
-242.505055,0.032817926,0.000645993
-247.3986605,0.031188869,0.000496448
-252.298335,0.032006535,0.000520398
-257.3160975,0.031702024,0.000572916
-262.3325575,0.031744513,0.000656471
-267.3482175,0.029565877,0.000658074
-272.363905,0.027907745,0.000555152
-277.3795425,0.021419505,0.001515462
-282.3948075,0.019496867,0.001064124
-287.4084475,0.012242374,0.001137212
-292.421485,0.006669651,0.000769452
-297.4379975,0.003084511,0.000419058
-302.4550575,0.002055262,0.00053601
-307.4712925,0.000547483,0.000117617
-312.4881275,0.000442644,0.000154365
-317.504815,-2.57E-05,7.59E-05
-322.52095,0.000402156,7.62E-05
-327.535635,0.000171301,0.000169722
-332.5509225,6.63E-05,0.000103988
-337.5677875,0.000357048,0.000233377
-342.586775,-0.000455311,0.000257242
-347.6062375,0.000117551,0.000105651
-352.6246275,0.000141132,6.95E-05
-357.64279,0.001402371,0.001084554
-362.5444842,0.002002406,0.001226431
+Time,MLR_mean,"Temp_avg, all4",MLR_error,Temperature_Error
```

Comment:
Even if you have quotes, do not use commas in column names for csv files.  Bad idea